sorc/checkout.sh

```diff
@@ -160,7 +160,7 @@ if [[ ${checkout_gsi} == "YES" ]]; then
 fi
 
 if [[ ${checkout_gdas} == "YES" ]]; then
-  checkout "gdas.cd" "https://github.com/NOAA-EMC/GDASApp.git" "1da2e63"; errs=$((errs + $?))
+  checkout "gdas.cd" "https://github.com/NOAA-EMC/GDASApp.git" "e61e33c"; errs=$((errs + $?))
```

Comment:
Sorry, I didn't catch this, but this hash needs to be also updated in `Externals.cfg`